### ISSUE TITLE
During the fs_fill test of the vfat file system, the size of the img does not match the number of cpus, resulting in a failure

### ISSUE THREAD

**Issue by hifilove** [linux-test-project/ltp#1036]
Environmental information:
```
# cat /proc/cpuinfo
skip...
processor       : 255
vendor_id       : AuthenticAMD
cpu family      : 25
model           : 1
model name      : AMD EPYC 7763 64-Core Processor
skip...
```

If there are many processors, an error will be reported:
```
fs_fill.c:100: TBROK: mkdir(mntpoint/thread257, 0700) failed: ENOSPC (28)
```
```
# lsblk
NAME                 MAJ:MIN RM  SIZE RO TYPE MOUNTPOINTS
loop0                  7:0    0  256M  0 loop /tmp/fs_5f4RX1/mntpoint
```

Because the size of the created image is 256M, the image will be created as fat16 by default when mkfs.vfat, and the maximum directory entry of fat16 is 512 (**will become fewer according to the length of the directory name**).

ltp/testcases/kernel/fs/fs_fill/fs_fill.c:
```
static void setup(void)
{
        unsigned int i;

        nthreads = tst_ncpus_conf() + 2;  //  more processors
        workers = SAFE_MALLOC(sizeof(struct worker) * nthreads);

        for (i = 0; i < nthreads; i++) {
                snprintf(workers[i].dir, sizeof(workers[i].dir),
                         MNTPOINT "/thread%i", i + 1);
                SAFE_MKDIR(workers[i].dir, 0700); // ENOSPC
        }

        tst_res(TINFO, "Running %i writer threads", nthreads);
}

```

**Comment by hifilove** [comment]
Maybe we can add parameters to mkfs to solve this problem.
```
MKFS.FAT(8)
skip...
       -F FAT-SIZE
           Specifies the type of file allocation tables used (12, 16 or 32 bit).  If nothing is specified, mkfs.fat will automatically select between 12, 16 and 32 bit, whatever fits better for the
           filesystem size.
skip...
```

**Comment by hifilove** [comment]
> Which LTP version are you using? This is supposedly fixed by [357d821](https://github.com/linux-test-project/ltp/commit/357d821741a72544d804aff2960aa3521b91767b)


The latest code is fine, thank you！
